src/utils/mounts.ts

```diff
@@ -118,6 +118,13 @@ async function mountDevice(
 
 // Unmounts device at path.  If the device is not mounted, this is a no-op.
 export async function unmountDevice(path: string) {
+  // Sometimes overlays are left over and need to be unmounted first.
+  const overlays = await findOverlayPaths(path)
+  for (const overlay of overlays) {
+    console.log(`Unmounting overlay ${overlay}`)
```

Comment:
Just in case anything's ever circular or w/e 🙂 

```suggestion
export async function unmountDevice(path: string, seenPaths = new Set<string>()) {
  if (seenPaths.has(path)) return
  seenPaths.add(path)

  // Sometimes overlays are left over and need to be unmounted first.
  const overlays = await findOverlayPaths(path, seenPaths)
  for (const overlay of overlays) {
    console.log(`Unmounting overlay ${overlay}`)
```